pkgs/development/web/netlify-cli/default.nix

```
@@ -1,12 +1,15 @@
 { callPackage, fetchFromGitHub, lib, pkgs }:
 let
   nodePackages = import ./composition.nix { inherit pkgs; };
+  meta = (lib.importJSON ./netlify-cli.json);
```

```suggestion
  sourceInfo = (lib.importJSON ./netlify-cli.json);
```

\+ below

`meta` already has a special meaning in packages.